RELEASE-NOTES.txt

```diff
@@ -3,6 +3,8 @@
 * [*] User Mention: Split the suggestions list into a prominent section and a regular section. [#19064]
 * [*] Use larger thumbnail previews for recommended themes during site creation [https://github.com/wordpress-mobile/WordPress-iOS/pull/18972]
 * [***] [internal] Block Editor: List block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/42702]
+* [**] Fix for Referrers Card Not Showing Search Engine Details [https://github.com/wordpress-mobile/WordPress-iOS/pull/19158]
+* [*] WeeklyRoundupBackgroundTask - format notification body [https://github.com/wordpress-mobile/WordPress-iOS/pull/19144]
```

Comment:
Just wanted to highlight the `WeeklyRoundupBackgroundTask` line. Should this be included?

Reply:
Thanks - I realised I had forgot it in my last PR so included it in release notes.  Think it needs a completely separate PR?  I just made a judgement call but can go either way

Reply:
IMO I think it's fine to keep it here since it was intentional. 👍